_posts/ericabrody/2016-04-05-MOAR-Dict-ericabrody.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+author: ericabrody
+title: "MOAR Dictionary"
+---
+
+Code for Exercise 1:
+<iframe src="https://trinket.io/embed/python3/933691312a" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Code for Exercises 2-5:
+<iframe src="https://trinket.io/embed/python3/f49bf4f86e" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+
+Reflection:
+Exercise 1 was challenging since I had gotten used to reading the files in line by line from the recent exercises and had forgotten how to use .read and putting all the words in the list. But I did feel confident in writing the code to add the items to the dictionary.
+I got stuck on Exercise 2 initially because I was looking for "From:" and didn't realize that the lines I was looking for didn't have the colon and the ones that did have the colon had only 2 elements in the line so when I was looking for the day of the week in item number 3 of the list of items in the line, there was no third item. It took a lot of print statements and patients to debug this and find the error. The rest of the exercises went ok, though I feel like I haven't been able to retain a lot of the methods (and their syntax) that we have learned from previous weeks as well as I would like.
```

Comment:
Print statements are your friend.

Retention of the particulars is not as important as being able to look up what you need.  If you'd like to help them sick, though, review the code from the last week or two and maybe a few of my solutions.  Literally read the code to yourself in english.  That may help it stick more.